docs/api/migration-guides/qiskit-runtime-setup.mdx

```diff
@@ -0,0 +1,301 @@
+```
+
+Additionally, you can now name your saved credentials and load the credentials by name.
+
+
+``` python
+# Save different accounts for open and premium access
+
+QiskitRuntimeService.save_account(channel="ibm_quantum", token="<IQX_TOKEN>", instance="h1/g1/p1", name="premium")
+QiskitRuntimeService.save_account(channel="ibm_quantum", token="<IQX_TOKEN>", instance="h2/g2/p2", name="open")
+
+# Load the "open" credentials
+
+service = QiskitRuntimeService(name="open")
+```
+  </TabItem>
+
+  <TabItem value="legacy" label="Legacy">
+``` python
+IBMQ.save_account("<IQX_TOKEN>", overwrite=True)
+```
+  </TabItem>
+</Tabs>
+
+
+## Load accounts
+
+Use the updated code to load accounts.
+
+<Tabs>
+  <TabItem value="updated" label="Updated">
+The new syntax combines the functionality from `load_account()` and
+`get_provider()` in one statement. The `channel` input parameter is
+optional. If multiple accounts have been saved in one device and no
+`channel` is provided, the default is `"ibm_cloud"`.
+
+``` python
+# To access saved credentials for the IBM cloud channel
+service = QiskitRuntimeService(channel="ibm_cloud")
+
+# To access saved credentials for the IBM quantum channel
+service = QiskitRuntimeService(channel="ibm_quantum")
+```
+  </TabItem>
+
+  <TabItem value="legacy" label="Legacy">
+``` python
+IBMQ.load_account()
+```
+  </TabItem>
+</Tabs>
+
+
+## Channel selection (get a provider)
+
+Use the updated code to select a channel.
+
+<Tabs>
+  <TabItem value="updated" label="Updated">
+The new syntax combines the functionality from `load_account()` and
+`get_provider()` in one statement. When using the `ibm_quantum` channel,
+the `hub`, `group`, and `project` are specified through the new
+`instance` keyword.
+
+``` python
+# To access saved credentials for the IBM quantum channel and select an instance
+service = QiskitRuntimeService(channel="ibm_quantum", instance="my_hub/my_group/my_project")
+```
+  </TabItem>
+
+  <TabItem value="legacy" label="Legacy">
+``` python
+provider = IBMQ.get_provider(project="my_project", group="my_group", hub="my_hub")
+```
+  </TabItem>
+</Tabs>
+
+## Get the system 
+
+Use the updated code to view systems and simulators.
+
+<Tabs>
+  <TabItem value="updated" label="Updated">
+``` python
+# You can specify the instance in service.backend() instead of initializing a new service
+backend = service.backend("ibm_backend", instance="h1/g1/p1")
+```
+If you don't know what backend you want to use, you can instead use code such as the following:
+
+```python
+from qiskit_ibm_runtime import QiskitRuntimeService
+
+service = QiskitRuntimeService()
+backend = service.least_busy(operational=True, simulator=False, min_num_qubits=num_qubits)
+```
+  </TabItem>
+
+  <TabItem value="legacy" label="Legacy">
+``` python
+provider = IBMQ.get_provider(hub="h1", group="g1", project="p1")
+backend = provider.get_backend("ibm_backend")
+```
+  </TabItem>
+</Tabs>
+
+## Upload, view, or delete custom prototype programs
```

Comment:
This can be moved to Overview (or common use cases) , since it's not specific to ibm[q]-provider.